index.html

```diff
@@ -0,0 +1,754 @@
+                            <li><a href="#">Jobs</a></li>
+                            <li><a href="#">Contact</a></li>
+                        </ul>
+                    </div>
+                    <div class="menu2 flex-spaceBetween">
+                        <ul class="menu-item2">
+                            <li><a href="#">LATEST</a></li>
+                            <li><a href="#">HARD FORK</a></li>
+                            <li><a href="#">PLUGGED</a></li>
+                            <li><a href="#">README</a></li>
+                            <li><a href="#">GROWTH QUARTERS</a></li>
+                            <li><a href="#">NEURAL</a></li>
+                        </ul>
+                        <ul>
+                            <li><i class="fas fa-search"></i></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>    
+
+            <div class="menuMobile">
+                <div class="mobileMenu flex-spaceBetween">
+                    <div><img src="assets/images/logo2.PNG" alt="logo-image"></div>
+                    <div>
+                        <i class="fas fa-search"></i>
+                        <i class="fas fa-bars"></i>
+                    </div>
+                </div>
+                <div class="weHeard">
+                    We heard you like events and sweatpants so we made event you
+                    can join in your seatpants <i class="fas fa-long-arrow-alt-right"></i>
+                </div>
+            </div>
+        </nav>
+    </header>
+
+    <section class="news-section flex">
+        <div class="news__article1">
+            <div class="feature-main__top">
+                <img src="assets/images/tnw.svg" alt="" class="feature-article__logo" />
+            </div>
+            <div class="title__bottom avalon">
+                <h3 class="news__title1">
+                    US Techs execs have dumped $3.6 billion in company stock this month
+                </h3>
+                <div class="news__author">
+                    by David Canellis <span><i class=" fas fa-clock"></i>6 hrs ago</span>
+                    <span><i class="fas fa-share-alt"></i>19</span>
+                </div>
+            </div>
+
+        </div>
+        <div class="news__article2 avalon">
+            <div class="news__title">
+                Shopify, Basecamp & more share what next for digital products on June 11
+            </div>
+        </div>
+        <div class="news__article3 avalon">
+            <div class="news__title">
+                Icon rewind is a time machine for your phone's app icons
+                <div class="news__author">
+                    by Max <span><i class=" fas fa-clock"></i>6 hrs ago</span>
+                    <span><i class="fas fa-share-alt"></i>11</span>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <main class="main">
+        <div class="latestNews">
```

Comment:
The `latest news` section should align to 3 articles per row in medium screen mode. Right now it continues with the styles for the desktop screens. It should follow the the [original site](https://thenextweb.com/)